package.json

```diff
@@ -53,6 +53,6 @@
     "lint": "eslint '{*.js,cmd/**/*.js,test/**/*.js}' --ignore-pattern=test/coverage",
     "test": "mocha",
     "test-cov": "istanbul cover --dir test/coverage _mocha && echo Coverage lies in test/coverage/lcov-report/index.html",
-    "test-travis": "istanbul cover _mocha --report lcovonly && cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js"
+    "test-travis": "istanbul cover --dir test/coverage _mocha --report lcovonly && cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js"
```

Comment:
I reverted the entire `package.json` file to what is on the master branch, thus this file should be identical? I will force push this file now.